examples/add_reverb_to_file.py

```diff
@@ -0,0 +1,135 @@
+"""
+Add reverb to an audio file using Pedalboard.
+
+The audio file is read in chunks, using nearly no memory.
+This should be one of the fastest possible ways to add reverb to a file.
+On my laptop, this runs about 32x faster than real-time
+(i.e.: processes a 60-second file in <2 seconds.)
+
+Requirements: `pip install pysoundfile tqdm pedalboard`
+"""
+
+import sys
+import warnings
+import argparse
+import numpy as np
+from tqdm import tqdm
+import soundfile as sf
+from tqdm.std import TqdmWarning
+from pedalboard import Reverb
+
+
+BUFFER_SIZE_SAMPLES = 1024
+NOISE_FLOOR = 1e-4
+
+
+def get_num_frames(f: sf.SoundFile) -> int:
+    # On some platforms and formats, f.frames == -1L.
+    # Check for this bug and work around it:
+    if f.frames > 2 ** 32:
+        f.seek(0)
+        last_position = f.tell()
+        while True:
+            # Seek through the file in chunks, returning
+            # if the file pointer stops advancing.
+            f.seek(1024 * 1024 * 1024, sf.SEEK_CUR)
+            new_position = f.tell()
+            if new_position == last_position:
+                return new_position
+            else:
+                last_position = new_position
+    else:
+        return f.frames
+
+
+def main():
+    warnings.filterwarnings("ignore", category=TqdmWarning)
+
+    parser = argparse.ArgumentParser(description="Add reverb to an audio file.")
+    parser.add_argument("input_file", help="The input file to add reverb to.")
+    parser.add_argument(
```

Comment:
Possible gotcha but should there be any checks to make sure the file doesn't exist/a flag to disable that or something like add _# to the file if the same file is found?